docs/src/index.md

```diff
@@ -23,7 +23,7 @@ To ensure a smooth learning experience, we have divided the workshop into three
 
 ## Pre-workshop preparation
 
-It is important that you install Julia and Visual Studio Code following the steps outlined in [Chapter 0](0_preparation/preparation.html).
+It is important that you install Julia and Visual Studio Code following the steps outlined in [Chapter 0](0_preparation/preparation).
```

Comment:
I believe this would be the correct way?
https://documenter.juliadocs.org/stable/man/guide/#Cross-Referencing

```suggestion
It is important that you install Julia and Visual Studio Code following the steps outlined in [Chapter 0](@ref).
```

Reply:
Lets see if it works before merging.

Reply:
It doesnt work with the suggested syntax.